moztelemetry/partition-packing/PartitionPerformance.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * This program is a benchmark test for running Spark Jobs on a cluster with different partition strategies.
+ */
+
+package com.mozilla.telemetry
+
+import com.mozilla.telemetry.heka.{Dataset}
+import org.apache.spark.sql.SparkSession
+import org.rogach.scallop.ScallopConf
+
+object PartitionPerformance {
+  def submitOnce(docType: String, channel: String, timestamp:String, minPartitions: Int) {
+    val startTime = System.currentTimeMillis()
+    val spark = SparkSession.builder.appName("ecole").getOrCreate()
+    implicit val sc = spark.sparkContext
+
+    val rdd = {
+      Dataset("telemetry")
+        .where("sourceName") {
+          case "telemetry" => true
+        }
+        .where("docType") {
+          case docType => true
+        }
+        .where("appUpdateChannel") {
+          case channel => true
+        }
+        .where("submissionDate") {
+          case timestamp => true
+        }
+        .records()
+    }
+    println(rdd.count())
```

Comment:
You should also print the number of partitions read via [`RDD.getNumPartitions` method](http://spark.apache.org/docs/latest/api/scala/index.html#org.apache.spark.rdd.RDD@getNumPartitions:Int).